src/quickstart/provider/minikube/boot.md

```diff
@@ -0,0 +1,48 @@
+# Booting Kubernetes Using Minikube
+
+This guide will walk you through the process of installing a small development
+Kubernetes cluster on your local machine using [minikube](https://github.com/kubernetes/minikube).
+
+## Pre-requisites
+
+* OS X
+    * [xhyve driver](https://github.com/kubernetes/minikube/blob/master/DRIVERS.md#xhyve-driver), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion) installation
+* Linux
+    * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [KVM](http://www.linux-kvm.org/) installation
+* Windows
+    * [Hyper-V](https://github.com/kubernetes/minikube/blob/master/DRIVERS.md#hyperv-driver)
+* VT-x/AMD-v virtualization must be enabled in BIOS
+* The most recent version of `kubectl`. You can install kubectl following
+  [these steps](http://kubernetes.io/docs/getting-started-guides/kubectl/).
```

Comment:
nit: link redirects to https://kubernetes.io/docs/user-guide/prereqs/ so perhaps use this instead?